expense-authorization/src/API/EMBC.ExpenseAuthorization.Api/Email/EmailRecipientService.cs

```diff
@@ -16,8 +17,8 @@ public EmailRecipientService(IOptions<EmailSettings> emailOptions)
         public IEnumerable<string> GetRecipients(ExpenseAuthorizationRequest request)
         {
             // based on the region, we may need to have different To addresses
-
-            yield return _emailOptions.Value.To;
+            var to = _emailOptions.Value.To ?? string.Empty;
+            return to.Split(new[] { ',', ';' }, StringSplitOptions.RemoveEmptyEntries));

```

Comment:
Or space-separated?

Reply:
no, just like in outlook

![image](https://user-images.githubusercontent.com/1844480/92149047-4bde5b00-edd2-11ea-91f2-0be88acfbbab.png)